index.html

```diff
@@ -61,20 +61,21 @@ <h3>CANOPY</h3>
         </div>
       </div>
 
-      <div class="cards">
+      <div class="cards right-image">
         <div class="img-project">
-          <img src="img/project-2.png" alt="Project number 2 image">
+          <img src="img/desktop-project-2.png" alt="Project number 2 image">
         </div>
         <div class="content">
           <h2>Multi-Post Stories</h2>
           <div class="info-tags">
-            <h3>CANOPY</h3>
-            <p>Back End Dev</p>
+            <h3>FACEBOOK</h3>
+            <p>Back Stack Dev</p>
             <p>2015</p>
           </div>
-          <p class="cards-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <p class="cards-p">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
           <ul class="tags">
             <li><a>HTML</a></li>
+            <li><a>Ruby on rails</a></li>
             <li><a>CSS</a></li>
             <li><a>JavaScript</a></li>
           </ul>
```

Comment:
Please, make sure you stick to the design as much as possible `(e.g., font, colors, images, text, margins)` using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
Kindly, I suggest you keep the mobile design exactly as it is in the Figma template when you update the Desktop version. 😄

> ## The Figma Design
>
><img width="179" alt="Screenshot 2023-06-08 at 5 07 30 PM" src="https://github.com/MasterLeiva/DavidLeiva-Portfolio/assets/91237525/b32e6774-5280-420a-84e9-0fa90a1910c0">
>
> ## Your Design
>
><img width="316" alt="Screenshot 2023-06-08 at 5 07 56 PM" src="https://github.com/MasterLeiva/DavidLeiva-Portfolio/assets/91237525/7576f8a7-0cef-4c19-96b8-6583dabdb8db">
><img width="312" alt="Screenshot 2023-06-08 at 5 08 58 PM" src="https://github.com/MasterLeiva/DavidLeiva-Portfolio/assets/91237525/97ce515d-af46-418d-87be-5f38bbf1dcbe">
><img width="316" alt="Screenshot 2023-06-08 at 5 10 27 PM" src="https://github.com/MasterLeiva/DavidLeiva-Portfolio/assets/91237525/cb9ec167-841e-4cdf-8c50-6a591f557e8d">